go.mod

```diff
@@ -28,7 +28,7 @@ require (
 	github.com/stretchr/testify v1.6.1
 	github.com/vishvananda/netlink v1.1.0
 	github.com/vishvananda/netns v0.0.0-20200728191858-db3c7e526aae // indirect
-	golang.org/x/net v0.0.0-20210420210106-798c2154c571 // indirect
```

Comment:
Just comment removal?

Reply:
It was automatically updated when running: `go mod tidy` and `go mod vendor`